assets/js/util/test/numFmt.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * `numFmt` tests
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { numFmt } from '../';
+
+/**
+ * Sets the Site Kit locale to a (legacy) global variable.
+ *
+ * @since 1.7.0
+ * @private
+ *
+ * @param {string} langCode The locale to set Site Kit to use. E.g. `en-US` or `de-DE`.
+ * @return {Object} Site Kit configuration object.
+ */
```

Comment:
I don't think we need to document internal test-only functions which aren't exported. Also the `@since` is inaccurate as it is 😄